src/tabs/osd.html

```diff
@@ -51,10 +51,8 @@ <h1 class="tab_title">
                             </div>
 
                             <div class="display-layout">
-                                <div class="col right">
                                     <div class="preview">
```

Comment:
👾

Reply:
I suppose the correct indentation is missing. I will fix.